### PR TITLE
cleanup enums to make them more consistent style wise

### PR DESCRIPTION
**Pull request by aronwk-aaron**
and use wise
Not touching dCommon yet, that's it's own special hell

no functional changes

**DO NOT SQUASH**

### CODE REVIEW

File: dCommon/dEnums/eMissionState.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	 * The mission is active and has been completed before. Used for daily missions.
+	 */
+	COMPLETE_ACTIVE = 10,
+
+	/**
+	* The mission has been completed before and has now been completed again. Used for daily missions.
+	*/
+	COMPLETE_READY_TO_COMPLETE = 12
+};
+
```
Comment (EmosewaMC) [inline dCommon/dEnums/eMissionState.h:55]: Formatting here a too nice.  Can we get some kind of staircase formatting here?

File: dGame/dInventory/Inventory.cpp [hunk 2]
```diff
@@ -231,41 +231,42 @@ eInventoryType Inventory::FindInventoryTypeForLot(const LOT lot) {
...
-	case eItemType::ITEM_TYPE_LEFT_TRINKET:
-	case eItemType::ITEM_TYPE_RIGHT_TRINKET:
-	case eItemType::ITEM_TYPE_COLLECTIBLE:
-	case eItemType::ITEM_TYPE_CONSUMABLE:
-	case eItemType::ITEM_TYPE_CHEST:
-	case eItemType::ITEM_TYPE_EGG:
-	case eItemType::ITEM_TYPE_PET_FOOD:
-	case eItemType::ITEM_TYPE_PET_INVENTORY_ITEM:
-	case eItemType::ITEM_TYPE_PACKAGE:
-	case eItemType::ITEM_TYPE_CURRENCY:
```
Comment (EmosewaMC) [inline dGame/dInventory/Inventory.cpp:264]: We should keep functional changes like this one out of this PR.

File: dGame/dInventory/Inventory.cpp [hunk 3]
```diff
@@ -231,41 +231,42 @@ eInventoryType Inventory::FindInventoryTypeForLot(const LOT lot) {
...
 
-	case eItemType::ITEM_TYPE_MODEL:
-	case eItemType::ITEM_TYPE_VEHICLE:
-	case eItemType::ITEM_TYPE_LOOT_MODEL:
-	case eItemType::ITEM_TYPE_MOUNT:
+	case eItemType::MODEL:
+	case eItemType::VEHICLE:
+	case eItemType::LOOT_MODEL:
+	case eItemType::LUP_MODEL:
+	case eItemType::MOUNT:
```
Comment (EmosewaMC) [inline dGame/dInventory/Inventory.cpp:247]: Same here

File: dGame/dMission/Mission.h [hunk 4]
```diff
@@ -10,8 +10,8 @@
 #include "MissionTask.h"
 #include "dCommonVars.h"
 #include "Entity.h"
-#include "MissionState.h"
-#include "MissionLockState.h"
+#include "eMissionState.h"
+#include "eMissionLockState.h"
```
Comment (EmosewaMC) [inline dGame/dMission/Mission.h:14]: These can be forward declared

File: dGame/Character.h [hunk 5]
```diff
@@ -9,7 +9,7 @@
 
 #include "NiPoint3.h"
 #include "NiQuaternion.h"
-#include "PermissionMap.h"
+#include "ePermissionMap.h"
```
Comment (EmosewaMC) [inline dGame/Character.h:12]: We can forward declare this as well

File: dGame/dMission/MissionTask.h [hunk 6]
```diff
@@ -4,7 +4,7 @@
 #define MISSIONTASK_H
 
 #include "CDMissionTasksTable.h"
-#include "MissionTaskType.h"
+#include "eMissionTaskType.h"
```
Comment (EmosewaMC) [inline dGame/dMission/MissionTask.h:7]: Forward declare

**Review by EmosewaMC — COMMENTED** [review]
5 comments and 1 funny

File: dGame/dInventory/Inventory.cpp (hunk 2 above)
Comment (aronwk-aaron) [reply dGame/dInventory/Inventory.cpp:264]: I would agree, but:
1. There's no items with that type, so there is essentially no functional change
2. It's correcting the type to be named correctly, which it's a model, so goes in models

Comment (aronwk-aaron) [reply dGame/dInventory/Inventory.cpp:264]: will revert change if you really want me to though

Comment (EmosewaMC) [reply dGame/dInventory/Inventory.cpp:264]: this change is fine since its a name correction and correcting the inventory an item of this type would go in

**Review by Jettford — APPROVED** [review]

**Review by EmosewaMC — APPROVED** [review]

**Pull request merged**